specs/commandline/sign.md

```diff
@@ -154,10 +152,10 @@ Warning: Always sign the artifact using digest(`@sha256:...`) rather than a tag(
 Successfully signed localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
 ```
 
-### Sign an artifact and store the signature using OCI image manifest
+### Sign an artifact and store the signature using OCI artifact manifest
```

Comment:
I think we can remove this usage.

Reply:
I think we can keep it for now and evaluate later since we still support this case.